projects/legacy/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -56,7 +56,7 @@ import {tuiExtractValueFromEvent} from './utils/extract-value-from-event';
 const MASK_SYMBOLS = /[ \-_()]/g;
 
 /**
- * @deprecated: drop in v5.0
+ * @deprecated: drop in v5.0 use {@link TuiInputPhoneInternational}
```

Comment:
```suggestion
 * @deprecated: drop in v5.0 use {@link TuiInputPhoneInternational} from @taiga-ui/kit
```

\+ add link to https://taiga-ui.dev/components/input-phone-international